src/CreeDictionary/API/search/eip_crk.py

```diff
@@ -0,0 +1,142 @@
+"""
+Cree-specific data for English Inflected Phrase search
+"""
+from morphodict.analysis.tag_map import TagMap
+
+
+def get_noun_tags(inflectional_category: str):
+    """Turn a wordform’s inflectional class into tags needed for FST generator.
+
+    >>> get_noun_tags("NA")
+    '+N+A'
+    >>> get_noun_tags("NA-3")
+    '+N+A'
+    >>> get_noun_tags("ND")
+    '+N+D'
+    """
+    noun_tags = ""
+    for c in inflectional_category:
+        if c == "-":
+            return noun_tags
+        noun_tags += "+" + c
+
+    return noun_tags
+
+
+# TagMaps can’t automatically pass through unknown tags because they wouldn’t
+# know how to order the unknown tags amongst the defined precedence classes. For
+# example, when populating a default tense tag of +Ind, it has to put that after
+# +V+TA and before +1Sg, so it needs precedence values for all of those tags.
+#
+# So we list everything.
+noun_passthrough_tags = {
+    0: [
+        # word class
+        "+N",
+        "+A",
+        "+I",
+        "+D",
+    ],
+    2: [
+        # N: Possessives
+        "+Px1Sg",
+        "+Px2Sg",
+        "+Px3Sg",
+        "+Px1Pl",
+        "+Px2Pl",
+        "+Px12Pl",
+        "+Px3Pl",
+        "+Px4Sg/Pl",
+        "+PxX",
+    ],
+    3: [
+        # N: number
+        "+Sg",
+        "+Pl",
+        "+Obv",
+        "+Loc",
+        "+Distr",
+    ],
+}
+
+verb_passthrough_tags = {
+    0: [
+        # word class
+        "+V",
+        "+TA",
+        "+AI",
+        "+II",
+        "+TI",
+    ],
+    2: [  # V: Person - subject
```

Comment:
For consistency with nearby comments

```suggestion
    2: [
      # V: Person - subject
```